src/index.html

```diff
@@ -7,8 +7,76 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#Apple"
+            class="is-active nav__link"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#Samsung"
+            class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
```

Comment:
```suggestion
          <li class="nav__item">
            <a href="#Apple"
            class="is-active nav__link"
          >
            Apple
            </a>
          </li>
          
          <li class="nav__item">
            <a href="#Samsung"
            class="nav__link"
          >
            Samsung
            </a>
          </li>
```

Add empty line between multiple sibling blocks of html